virttest/utils_misc.py

```diff
@@ -2791,7 +2791,7 @@ def format_windows_disk(session, did, mountpoint=None, size=None,
             logging.debug("Detail for 'Disk%s'" % did)
             details = session.cmd_output(detail_cmd)
 
-            if re.search("Status.*Offline", details, re.I | re.M):
+            if re.search("Disk.*Offline", details, re.I | re.M):
```

Comment:
Is that the output contains this pattern (`Disk.*Offline`) on all windows platform?